frontend/templates/404.html

```diff
@@ -1,13 +1,17 @@
 {% extends "base.html" %}
 
+{% from "partials/contact_details.html" import contact_details %}
+
 {% block pageTitle %}Page not found – {{ service_title }}{% endblock %}
 
 {% block content %}
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Page not found</h1>
-    <p class="govuk-body">If you typed the web address, check it is correct.</p>
-    <p class="govuk-body">If you pasted the web address, check you copied the entire address.</p>
-  </div>
-</div>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
+            <p class="govuk-body">Try again later.</p>
+            <p class="govuk-body">If the web address is correct, or you selected a link or button, contact the Community Ownership Fund team
+                to speak to an adviser. </p>
```

Comment:
Do we want to say 'speak to an advisor'? We might only be giving them an email address not a phone number